### DISCUSSION TITLE
How to display an image with a radius, just like a round avatar

### DISCUSSION THREAD

**Discussion by kirito41dd** [slint-ui/slint#3321] [Q&A] [answered]
As the title

**Comment by tronical** [comment, marked as answer]
You could wrap the `Image` in a `Rectangle` that has clipping enabled and a border-radius set. A bit like in [this](https://slintpad.com/?snippet=import+%7B+Button%2C+VerticalBox+%7D+from+%22std-widgets.slint%22%3B%0Aexport+component+Demo+%7B%0A++++VerticalBox+%7B%0A++++++++alignment%3A+start%3B%0A++++++++Text+%7B%0A++++++++++++text%3A+%22Hello+World%21%22%3B%0A++++++++++++font-size%3A+24px%3B%0A++++++++++++horizontal-alignment%3A+center%3B%0A++++++++%7D%0A++++++++Rectangle+%7B%0A++++++++++++clip%3A+true%3B%0A++++++++++++width%3A+self.height%3B%0A++++++++++++border-radius%3A+self.height%3B%0A++++++++++++Image+%7B%0A++++++++++++++++source%3A+%40image-url%28%22https%3A%2F%2Fslint.dev%2Flogo%2Fslint-logo-full-light.svg%22%29%3B%0A++++++++++++++++height%3A+100px%3B%0A++++++++++++%7D%0A++++++++%7D%0A++++++++HorizontalLayout+%7B+alignment%3A+center%3B+Button+%7B+text%3A+%22OK%21%22%3B+%7D+%7D%0A++++%7D%0A%7D%0A) example. Is that what you mean?
